### PR TITLE
Create B827EBFFFEDDF5E8.json

### PR DESCRIPTION
**Pull request by ahytaipei**
I am trying to follow the tutorial in https://www.hackster.io/naresh-krish/getting-started-with-the-rak-831-lora-gateway-and-rpi3-e3351d
Anyway, in this file I am not sure about the ref_altitude

### CODE REVIEW

File: B827EBFFFEDDF5E8.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFEDDF5E8", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-asia-se",
```
Comment (gonzalocasas) [inline B827EBFFFEDDF5E8.json]: This is not the correct URL for the router in Asia.

Comment (ahytaipei) [reply B827EBFFFEDDF5E8.json]: I have updated it. Anyway I am also new to Git. Can you see my recent edits? Thanks

File: B827EBFFFEDDF5E8.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFEDDF5E8", 
+		"servers": [ 
+			{ 
+				"server_address": "router.asia-se.thethings.network",
```
Comment (gonzalocasas) [inline B827EBFFFEDDF5E8.json]: This is still the wrong URL. Please check here for the list of routers: https://www.thethingsnetwork.org/wiki/Backend/Connect/Gateway

Comment (ahytaipei) [reply B827EBFFFEDDF5E8.json]: I have updated the file. Please kindly check it. Thank you.

**Comment by gonzalocasas** [comment]
@ahytaipei Perfect! Thanks! It's now merged.

**Pull request merged**